src/test/handler.test.ts

```diff
@@ -29,47 +36,103 @@ describe('getParamsFromUriPath', () => {
 
     it('return empty file if document has no file format', () =>
         expect(getParamsFromUriPath('git://github.com/sourcegraph/sourcegraph/testrepo#formatless')).toEqual({
-            repo: 'sourcegraph/sourcegraph',
+            repo: 'sourcegraph/sourcegraph/testrepo#formatless',
             file: null,
         }))
 })
 
-const paramsWeb = {
-    repo: 'sourcegraph/sourcegraph',
-    file: '#web/src/storm/index.tsx',
-}
-
-export const paramsDev = {
-    repo: 'sourcegraph/dev-repo',
-    file: '#dev/backend/main.go',
-}
-
-export const paramsNone = {
-    repo: 'sourcegraph/test-repo',
-    file: '#dev/test/start.rb',
-}
+const paramsInput = [
+    {
+        goal: 'returns a web project that matches the repo and file patterns',
+        params: {
+            repo: 'sourcegraph/sourcegraph',
+            file: '#web/src/storm/index.tsx',
+        },
+        expected: projects[0],
+    },
+    {
+        goal: 'returns a dev project that matches the repo and file patterns',
+        params: {
+            repo: 'sourcegraph/dev-repo',
+            file: '#dev/backend/main.go',
+        },
+        expected: projects[1],
+    },
+    {
+        goal: 'returns undefined for not matching repo and file patterns',
+        params: {
+            repo: 'sourcegraph/test-repo',
+            file: '#dev/test/start.rb',
+        },
+        expected: undefined,
+    },
+    {
+        goal: 'returns undefined for not matching repo and file patterns',
+        params: {
+            repo: 'sourcegraph/test-repo',
+            file: '#dev/test/start.rb',
+        },
+        expected: undefined,
+    },
+]
 
 describe('matchSentryProject', () => {
-    it('returns a web project that matches the repo and file patterns', () =>
-        expect(matchSentryProject(paramsWeb, projects)).toEqual(projects[0]))
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
+    for (const paramsCase of paramsInput) {
+        it('fullfils the following goal: ' + paramsCase.goal, () =>
```

Comment:
The prefix doesn't add anything here
```suggestion
        it(paramsCase.goal, () =>
```